azure-pipelines.yml

```diff
@@ -36,14 +36,14 @@ jobs:
         DEVITO_BACKEND: 'core'
         RUN_EXAMPLES: 'true'
         PYTHON_VERSION: '3.7'
-      nompi-conda-gcc5:
+      nompi-conda-gcc5-py38:
         gccVersion: '5'
-        DEVITO_OPENMP: '0'
+        DEVITO_OPENMP: '1'
```

Comment:
why this one?

Reply:
No openmp already tested no reason to not use it

Reply:
makes it slower?

Reply:
That build is ~10min faster tan before

Reply:
thanks to openmp?

Reply:
Probably partially yes, all azure instances have at least 2 CPU